services/Errand/dna/@types/holochain/holochain.d.ts

```diff
@@ -0,0 +1,82 @@
+// holochain type definitions
+
```

Comment:
This is the old version of the types. The new one is installed as an NPM dependency and we should move over to using that when possible.